pegasus/sites.v3/code.org/layouts/wide.haml

```diff
@@ -1,10 +1,5 @@
 .wrapper
-  -if header['theme'] == 'responsive_wide'
-    #desktop-header.desktop-feature
-      -#= view :header
-    #mobile-header.mobile-headers.mobile-feature
-      -#= view :mobile_header_responsive
-  -else
+  -unless header['theme'] == 'responsive_wide'
```

Comment:
These css classes are still used later in the same layout and also in the default layout.

I don't think there's really any other way to simplify it. Different parts will show if theme is responsive_wide, some won't, and some will show no matter the theme. We can't just put one gate at the beginning since the order matters.